common/ux-alignment.cpp

```diff
@@ -4,6 +4,7 @@
 #include "ux-alignment.h"
 #include <imgui.h>
 #include <imgui_impl_glfw.h>
+#include <imgui_impl_opengl3.h>
```

Comment:
Is this new that we must use opengl3 and not only glfw?

Reply:
There was a compile issue only using glfw that was solved only by adding opengl3 (based on the ImGui [getting started guide](https://github.com/ocornut/imgui/wiki/Getting-Started#using-another-combination-of-backends))